test/unit/app/utils/test_user.py

```diff
@@ -4,17 +4,18 @@
 import pytest
 
 
-def test_create_transfer_account_user(create_transfer_account_user):
-    """
-    GIVEN create_transfer_account_user function
-    WHEN called with first_name, last_name, phone
-    THEN assert one_time_code, transfer account exists,
-        transfer_account approval is equal to config
-    """
-    import config
-    assert create_transfer_account_user.one_time_code is not None
-    assert create_transfer_account_user.transfer_account is not None
-    assert create_transfer_account_user.transfer_account.is_approved is config.AUTO_APPROVE_TRANSFER_ACCOUNTS
+# REDACTED: USERS NOW HAVE MULTIPLE TRANSFER ACCOUNTS
```

Comment:
this whole test doesn't really work on the new system because there's now multiple transfer accounts per user, though we should probably re-create a new equivalent test